### ISSUE TITLE
apps.openerp.com is broken

### ISSUE THREAD

**Issue by ypapouin** [odoo/odoo#83402]
https://apps.openerp.com is an alias for https://apps.odoo.com but is totally broken.

It is impossible to download any modules

eg: Open https://apps.openerp.com/apps/modules/12.0/w_disable_debug_mode/
Clicking on `Download For v 12.0` doesn't work correctly

By the way, I tried to use the "Contact Us" page to warn you but the form is broken too with both domain ...
![image](https://user-images.githubusercontent.com/22446243/151150504-d6ee221a-01ae-4f48-97ac-ba969bc3df96.png)
![image](https://user-images.githubusercontent.com/22446243/151150230-70ce97f6-e270-445d-a1a8-186c7dbd62d7.png)

**Comment by mattybell** [comment]
Try this link instead, I think the devs didn't update the download link (they're still on openerp) 
https://apps.odoo.com/apps/modules/12.0/w_disable_debug_mode/#

**Comment by Yenthe666** [comment]
Why would you even use openerp.com if you have odoo.com though?

**Comment by ypapouin** [comment]
Ask google SEO :) or maybe reformulate to: Why not just redirecting openerp.com to odoo.com ?

![image](https://user-images.githubusercontent.com/22446243/157662740-c14040ad-b845-4bf7-a180-7b6cedce0220.png)

**Comment by Yenthe666** [comment]
Yeah it should redirect for sure :) @mart-e maybe you can give a hand here?

**Comment by xavieralt** [comment]
@ypapouin @Yenthe666 the downloads from `apps.openerp.com` should works again ;) (cc @mart-e)
